Plugin.cs

```diff
@@ -1,39 +1,64 @@
-﻿using System;
-using BepInEx;
+﻿using BepInEx;
 using BepInEx.Logging;
+using BepInEx.Configuration;
 using HarmonyLib;
 using LCAlwaysHearWalkieMod.Patches;
-using UnityEngine.Scripting;
 
-namespace BepInEx5.PluginTemplate
+namespace LCAlwaysHearWalkieMod;
+
+public static class PluginInfo
 {
-    [BepInPlugin(modGUID, modName, modVersion)]
-    public class LCAlwaysHearWalkieMod : BaseUnityPlugin
+    public const string modGUID = "suskitech.LCAlwaysHearActiveWalkie";
+    public const string modName = "LC Always Hear Active Walkies";
+    public const string modVersion = "1.4.2";
+}
+
+[BepInPlugin(PluginInfo.modGUID, PluginInfo.modName, PluginInfo.modVersion)]
+public class Plugin : BaseUnityPlugin
+{
+    private ConfigEntry<float> configAudibleDistance;
```

Comment:
I'm not familiar with configs in bepinex. I'm assuming [this](https://docs.bepinex.dev/articles/user_guide/configuration.html) is the correct documentation for it? I'll need to make sure to update the readme with details on how to configure the mod.